cypress/integration/unit/user-organisation-management.spec.js

```diff
@@ -144,7 +144,7 @@ context('testing user and organisation management', () => {
     });
 
     it('check filter by group', () => {
-      checkRoleFilterSingle('Admin', 8);
+      checkRoleFilterSingle('Admin', 9);
       checkRoleFilterSingle('Secretarie', 11);
```

Comment:
I think all of these should be "van 12", since we are filtering out one by one
I'll check locally to avoid multiple jenkins runs